flake.nix

```diff
@@ -13,7 +14,7 @@
         nix-serve = with final; stdenv.mkDerivation {
           name = "nix-serve-${self.lastModifiedDate}";
 
-          buildInputs = [ perl nix.perl-bindings perlPackages.Plack perlPackages.Starman perlPackages.DBDSQLite ];
+          buildInputs = [ perl nix.defaultPackage.${super.system}.passthru.perl-bindings perlPackages.Plack perlPackages.Starman perlPackages.DBDSQLite ];
```

Comment:
it doesn't seem like `super.system` is correct. I suppose you meant `prev.system`?